lambda/call-insights-parsing/docker/app.py

```diff
@@ -0,0 +1,137 @@
+import json
```

Comment:
Just curious. Any reason to use python over node? Most of our team are more familiar with Node :D